src/v2/view-builder/views/webauthn/ChallengeWebauthnView.js

```diff
@@ -115,8 +148,93 @@ const Body = BaseForm.extend({
   }
 });
 
+const Footer = AuthenticatorFooter.extend({
```

Comment:
Lets move out this logic to a separate file like this 

https://github.com/okta/okta-signin-widget/blob/master/src/v2/view-builder/components/IdentifierFooter.js

May be not under shared